docs/dev-guide/src/config/flags.md

```diff
@@ -192,6 +193,10 @@ When enabled, UUIDs of expressions and specifications printed with [`PRINT_TYPEC
 
 When enabled, debug files dumped by `rustc` will not contain lifetime regions.
 
+## `INTERNAL_ERRORS_AS_WARNINGS`
+
+When enabled, internal errors are presented as warnings.
```

Comment:
Maybe let's add something to make it clear this is also a debugging flag, because internal errors could hide an actual verification error.